pp_hot.c

```diff
@@ -3938,12 +3972,21 @@ PP(pp_iter)
     case CXt_LOOP_LAZYIV: /* integer increment */
     {
         IV cur = cx->blk_loop.state_u.lazyiv.cur;
-        if (UNLIKELY(cur > cx->blk_loop.state_u.lazyiv.end))
+            bool pad_it = FALSE;
+            if (UNLIKELY(cur > cx->blk_loop.state_u.lazyiv.end)) {
+                if (LIKELY(!i)) {
             goto retno;
+                }
```

Comment:
There's more confusing white spacing here

Reply:
As @tonycoz spotted (below, on this view) it's a side effect of me not realising that the `git rebase` munged my whitespace. "Fixed in post" - er, on the upcoming rebase.